release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -1156,6 +1156,29 @@ The Thanos Querier aggregates and optionally deduplicates core {product-title} m
 
 The Virtual Machine `Pending Changes` alert is now more informative. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1862801[*BZ#1862801*])
 
+[id="ocp-4-6-insights-operator"]
+=== Insights Operator
+
+[id="ocp-4-6-insights-operator-data-collection-enhancements”]
+==== Insights Operator data collection enhancements
+
+In {product-title} 4.6, the Insights Operator collects the following additional information:
+
+* Pod disruption budgets
+* The volume snapshot custom resource definition
+* The latest pod logs from pods that are not healthy
+* Running image data, including the number of containers using an image and the age of the corresponding pods
+* JSON dumps for pods that have crash looping containers
+* The MachineSet resource configuration
+* The anonymized HostSubnet resource configuration
+* The MachineConfigPools configuration
+* The InstallPlans and their count
+* ServiceAccounts statistics from Openshift namespaces
+
+Additionally, with this release the Insights Operator collects information about all cluster nodes, while previous versions only collected information about unhealthy nodes.
+
+With this additional information, Red Hat can provide improved remediation steps in {cloud-redhat-com}.
```

Comment:
This should specifically point to cloud.redhat.com/openshift or talk about OpenShift Cluster Manager.

Reply:
@radekvokal thank you for your review. The `{cloud-redhat-com}` attribute renders as 'Red Hat OpenShift Cluster Manager' in the docs build. In the published document, the line will read as follows:

"With this additional information, Red Hat can provide improved remediation steps in Red Hat OpenShift Cluster Manager."